build.gradle

```diff
@@ -208,6 +208,9 @@ subprojects {
     version = rootProject.version
 
     repositories {
+        if (project.hasProperty("enableMavenLocal")) { // useful for development
```

Comment:
This might be potentially dangerous, as there might be local artifacts even in `.m2` used by the build pipeline. I'd not keep it. Besides, if local libs are required, developer can always set this up easily during local development and then not commit it. I fear the side effects. Artifacts could be overridden locally in worst case....